frontend/src/components/models.ts

```diff
@@ -151,7 +151,6 @@ export type RelayResponse = { relayTransactionHash: string } | ApiError;
 export type RelayerStatusResponse =
   | { receivedTime: string; broadcasts?: any[]; receipt?: TransactionReceipt }
   | ApiError;
-export type ConfirmedRelayerStatusResponse = { receivedTime: string; broadcasts: any[]; receipt: TransactionReceipt };
```

Comment:
_(arbitrary comment location to start a thread)_

> Must be merged and deployed at the same time as https://github.com/ScopeLift/umbra-api/pull/54

Given this, how do I test the new behavior? I don't think I've ever tested tested out the app using a localhost backend, so instructions for testing would be helpful!

Reply:
Good question. I've added [a commit](https://github.com/ScopeLift/umbra-api/pull/54/commits/d22c942ed373e8fa7dbcf326e5bfeaaaffc185ed) to the API PR that explains how to do this.